### ISSUE TITLE
pacman to deb issue

### ISSUE THREAD

**Issue by jjgalvez** [jordansissel/fpm#1530]
when converting a pacman file to a deb file the resulting deb file contained the .BUILDINFO file from the pacman build which was then inappropriately copied to root.  I don't think this file should be included in the resultant build.

**Comment by futpib** [comment]
Workaround I'm using:

```bash
fpm -s pacman -x .BUILDINFO -x .PKGINFO -x .MTREE -t deb *.pkg.tar.xz
```

**Comment by jordansissel** [comment]
Agreed this is a bug. Thanks for the clear workaround! It should make
fixing this in FPM easier.
